task/init/0.2/init.yaml

```diff
@@ -25,7 +25,7 @@ spec:
 
   steps:
     - name: init
-      image: registry.redhat.io/openshift4/ose-cli:4.13@sha256:73df37794ffff7de1101016c23dc623e4990810390ebdabcbbfa065214352c7c
+      image: quay.io/openshift/origin-cli@sha256:086156148c5ef531223b959c3636f0cdb556272eb5a181a3a2fe43d822f41b4f
```

Comment:
We could instead rework the `init` task to use `skopeo` rather than `oc image`, and then we could use the registry.access.redhat.com skopeo image

Reply:
great idea! do you have the full reference for this skopeo image?

Reply:
In the apply-tags task we use `registry.access.redhat.com/ubi9/skopeo:9.4-6@sha256:c4d70dec3eb0a0c831490192145ea25431fe04d1cf307f8d61e2d87adb41e7e3`